src/subscription_manager/cli_command/org.py

```diff
@@ -46,18 +46,19 @@ def _get_org(org):
     @property
     def org(self):
         if not self._org:
-            owners = self.cp.getOwnerList(self.options.username)
-            if len(owners) == 1:
-                self._org = owners[0]['key']
-            elif self.options.org is None:
-                # Get a list of valid owners. Since no owner was specified,
-                # print a hint message showing available owners, before asking
-                # to enter one.
-                org_keys = [owner['key'] for owner in owners]
-                print(_(
-                    'Hint: User "{name}" is member of following organizations: {orgs}'
-                ).format(name=self.username, orgs=', '.join(org_keys)))
-                self._org = self._get_org(self.options.org)
-            else:
+            if self.options.org is not None:
                 self._org = self.options.org
+            else:
+                owners = self.cp.getOwnerList(self.options.username)
+                if len(owners) == 1:
+                    self._org = owners[0]['key']
+                else:
```

Comment:
This is not 100% correct. When user is not member of any organization (yes, it is possible), then behavior of sub-man is wrong in this case:

```
[root@thinkpad-t580 subscription-manager]# PYTHONPATH=./src python -m \
    subscription_manager.scripts.subscription_manager \
    syspurpose role --list --username magoo --password password
Hint: User "magoo" is member of following organizations: 
Organization:
```

I believe that `subscription-manager` should be terminated with some error message (e.g.: 'User "magoo" is not member of any organization') and non-zero exit code should be returned.

BTW: When magoo user tries to register, then we handle this situation gracefully.

Reply:
Thanks for the mention of the corner case situation. Since it was already a problem in the existing code, I added an extra commit to gracefully handle the situation better with an informative (I hope) error message.